eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/serialization/core/Box.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.serialization.core;
+
+import android.os.Parcelable;
+
+
+/**
+ * @author Gabor Keszthelyi
+ */
+public interface Box<T> extends Parcelable
+{
+
+    T content();
```

Comment:
ah, scratch that. It would still not have a `Context`. So that's not an option. Ok I guess we can't do this then.